buffer/src/test/java/io/netty/buffer/AbstractByteBufTest.java

```diff
@@ -2268,6 +2268,7 @@ private void testBytesInArrayMultipleThreads(final boolean slice) throws Excepti
         final CyclicBarrier barrier = new CyclicBarrier(11);
         for (int i = 0; i < 10; i++) {
             new Thread(new Runnable() {
+                @SuppressWarnings("MisorderedAssertEqualsArguments")
```

Comment:
which line(s) does this apply to? Based upon [1] it is not clear which line would cause this warning.

[1] https://searchcode.com/codesearch/view/7358059/